test/server/http/admin_test.cc

```diff
@@ -293,5 +294,105 @@ TEST(PrometheusStatsFormatter, FormattedTags) {
   EXPECT_EQ(expected, actual);
 }
 
+TEST(PrometheusStatsFormatter, statsAsPrometheus) {
+
+  // create two clusters and two gauges with metric name being same same and
```

Comment:
nit: I would suggest that you change the comment to the following (similar for the comment in the second test):
```
Create two counters and two gauges with each pair having the same name, but having different tag names and values.  `statsAsPrometheus()` should return two implying it found two unique stat names.